packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -122,21 +128,24 @@ export async function checkAccountExists(userEmail: string) {
 export async function emailInputValidationAndAccountCheck(
   value: string,
   focused: boolean,
+  setValidEmail: Function,
+  setAccountExists: Function,
   setEmailInputState: Function,
-  getString: Function,
-  checkAccountExists: Function
+  checkAccountExists: Function,
```

Comment:
I see that you defined the type at https://github.com/mozilla/fxa/pull/10009/files#diff-e7d37441df07a444329b2215c48e3c74a7a1c5d26f586cea6958da8bb385a9a8R22 Why not use it here?  `setValidEmail` and `setAccountExists` too.